pkg/generate/generate.go

```diff
@@ -173,18 +173,15 @@ func (c *Controller) applyGeneratePolicy(log logr.Logger, policyContext engine.P
 
 		processExisting := false
 
-		for _, v := range policy.Spec.Rules {
-			if policy.Name == gr.Spec.Policy {
-				if len(v.MatchResources.ResourceDescription.Kinds) > 0 && (len(v.MatchResources.ResourceDescription.Annotations) == 0 || len(v.MatchResources.ResourceDescription.Selector.MatchLabels) == 0) {
-					processExisting = func() bool {
-						rcreationTime := resource.GetCreationTimestamp()
-						pcreationTime := policy.GetCreationTimestamp()
-						return rcreationTime.Before(&pcreationTime)
-					}()
-				}
+		if len(rule.MatchResources.Kinds) > 0 {
+			if len(rule.MatchResources.Annotations) > 0 || rule.MatchResources.Selector == nil {
```

Comment:
Same here - `v.MatchResources.Selector != nil`?

Reply:
According to current logic, when processExisting=true, we would not apply this generate policy (the name `processExisting` is confusing here, when I see processExisting = true, I would expect the policy to be applied to the existing resource).

 Again, in order to trigger the application of a generate policy, we need a matched annotation or label selector. I believe here it should also be `len(rule.MatchResources.Annotations) == 0`.